include/rtdef.h

```diff
@@ -1074,7 +1076,7 @@ struct rt_device_graphic_info
 {
     rt_uint8_t  pixel_format;                           /**< graphic format */
     rt_uint8_t  bits_per_pixel;                         /**< bits per pixel */
-    rt_uint16_t reserved;                               /**< reserved field */
+    rt_uint16_t pitch;                                  /**< bytes per line */
```

Comment:
why not use bytes_per_line, which is more explict.

Reply:
The `pitch` is more clear in graphic.